servicemodules/discord.py

```diff
@@ -105,13 +102,7 @@ async def on_message(message: discord.message.Message) -> None:
                 try:
                     response = module.process_message(message)
                 except Exception as e:
-                    parts = ["Traceback (most recent call last):\n"]
-                    parts.extend(traceback.format_stack(limit=25)[:-2])
-                    parts.extend(traceback.format_exception(*sys.exc_info())[1:])
-                    error_message = "".join(parts)
-                    if self.error_channel is None:
-                        self.error_channel = self.utils.client.get_channel(error_channel_id)
-                    await self.error_channel.send(f"```{error_message}```")
+                    await self.utils.log_exception(e)
```

Comment:
is there any disadvantage for calling `process_message()` without any try/except and let our new default exception handling mechanism to log anything unexpected?

Reply:
What is our new default exception handling mechanism?

Reply:
hmm, I'm checking PR #159 again and that's just for Discord, my memory was too optimistic that it was more general for all services 🤔